src/app/shared/services/record.service.ts

```diff
@@ -41,4 +48,8 @@ export class RecordService {
       .map(res => res.json());
   }
 
+  saveRecord(record: Object): Observable<Object> {
+    record['control_number'] =  parseInt(record['control_number'], 10);
```

Comment:
Because the schema validation failed when submitted the PUT request . `control_number` is of `type : integer` in my inspire-schemas `hep.json`.